package.json

```diff
@@ -28,6 +28,20 @@
     "main": "dist/main/js/main.js",
     "types": "dist/main/js/main.d.ts",
     "unpkg": "dist/imsc.min.js",
+    "exports": {
+        ".": {
+            "types": "./dist/index.d.ts",
+            "import": "./dist/index.js"
+        },
+        "./*.js": {
+            "types": "./dist/*.d.ts",
+            "import": "./dist/*.js"
+        },
+        "./*": {
+            "types": "./dist/*.d.ts",
+            "import": "./dist/*.js"
+        }
+    },
```

Comment:
Added granular file imports for anyone installing `imscJs` via npm. This way `fromXML` can be imported directly, and avoid bundling `sax` unless `createSAXParser` is explicitly imported.